lib/graphql/subscriptions/event.rb

```diff
@@ -53,6 +53,34 @@ def fingerprint
 
       class << self
         private
+
+        # This method does not support cyclic references in the Hash,
+        # nor does it support Hashes whose keys are not sortable
+        # with respect to their peers ( cases where a <=> b might throw an error )
+        def deep_sort_hash_keys(hash_to_sort)
+          raise ArgumentError("Argument must be a Hash") unless hash_to_sort.is_a?(Hash)
```

Comment:
```suggestion
          raise ArgumentError.new("Argument must be a Hash") unless hash_to_sort.is_a?(Hash)
```

(I don't think this will do what you expect:

```
ArgumentError("boom!")
# (irb):1:in `<main>': undefined method `ArgumentError' for main:Object (NoMethodError)
#	from /Users/rmosolgo/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/irb-1.3.5/exe/irb:11:in `<top (required)>'
#	from /Users/rmosolgo/.rbenv/versions/3.0.2/bin/irb:23:in `load'
#	from /Users/rmosolgo/.rbenv/versions/3.0.2/bin/irb:23:in `<main>'
ArgumentError.new("boom!")
# => #<ArgumentError: boom!>
```

)